etc/apache-mod_proxy_fcgi.conf.template

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
I'd probably just do it here unless you think it's going to be disruptive. Only impact should be on `web.py`, though, no?